### ISSUE TITLE
[Bug]: eslint/eslint_d duplicated messages

### ISSUE THREAD

**Issue by andreicorpo** [LunarVim/LunarVim#1931]
### Problem description

Steps:
- Open a blank javascript file in a folder configured to run `eslint` or `eslint_d`.
- Write some code which should errors/warnings
- run 
``` 
lua require"lvim.lsp.handlers".show_line_diagnostics()
``` 
or 
```
lua vim.lsp.diagnostic.show_line_diagnostic()
```

You will then see that the same message appears multiple times.

### Check the following

- [X] I am in a git managed directory
- [X] I restarted Neovim after running :LspInstall
- [X] I have a valid language configuration (~/.config/lvim/ftplugin/)
- [X] I checked the options (~/.config/lvim/config.lua)
- [X] The programs specified by the config are installed

### LunarVim version

084e74f8

### Neovim version (>= 0.5)

NVIM v0.6.0-dev+nightly-936-g1fdbd29df

### Screenshots

![image](https://user-images.githubusercontent.com/12585256/140770011-0ad6686d-4d2b-4c0e-9966-01e52003ba14.png)

![image](https://user-images.githubusercontent.com/12585256/140771118-9e41a4d4-4e67-46d8-9084-a16e11cffe58.png)

![image](https://user-images.githubusercontent.com/12585256/140771163-8339b647-1517-4957-b4da-7e0797708b30.png)

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by kylo252** [comment]
Hmm, this was supposed to be fixed by 5e9b1c817f68d345f1933b13638b5766709a8b6d.

**Comment by andreicorpo** [comment]
It seems like I don't get anymore duplicate messages using `lua require"lvim.lsp.handlers".show_line_diagnostics()`, even tho I didn't do anything. 🤔 . But I still do get duplicate messages if I run `lua vim.lsp.diagnostic.show_line_diagnostic()` - it makes me think that the linter somehow runs multiple times per file? 🤔

**Comment by kylo252** [comment]
> It seems like I don't get anymore duplicate messages using `lua require"lvim.lsp.handlers".show_line_diagnostics()`, even tho I didn't do anything. 🤔 . But I still do get duplicate messages if I run `lua vim.lsp.diagnostic.show_line_diagnostic()` - it makes me think that the linter somehow runs multiple times per file? 🤔

It's because we have a custom handler that shows the error code and source, but it's not really compatible with the new handlers from upstream. There was an attempt at mitigating that, but I guess it's not really that robust.

Can you see if #1934 fixes your problem?

**Comment by ChristianChiarulli** [comment]
I patched this issue here: https://github.com/LunarVim/LunarVim/commit/5e9b1c817f68d345f1933b13638b5766709a8b6d

Although I'm sure there is a more elegant solution.

**Comment by chase** [comment]
Since this was patched, should we close this @ChristianChiarulli ?

**Comment by ChristianChiarulli** [comment]
@chase we can leave it open for now. It will remind me to look into the real issue at some point.

**Comment by kylo252** [comment]
This is now fixed on rolling after #2037
